docs/elastic-agent-gke-autopilot.md

```diff
@@ -0,0 +1,236 @@
+
+![Agent](./images/3.png)
+
+**Use Manifests of next Step 4 to complete installation**
+
+4. **Step 4: Edit provided Agent Manifest**
+
+Depending on the Elastc agent mode chosen, pick relevant manifest below:
+
+1. [Elastic Agent Managed Manifest](./manifests/elastic-agent-managed-gke-autopilot.yaml)
+
+- Set Container Image version of Elastic Agent
+
+```yaml
+containers:
+    - name: elastic-agent-standalone
+      image: docker.elastic.co/beats/elastic-agent:8.5.3
+```
+
+- Provide Elastic Cloud Credentials (In managed scenario you only neeed `FLEET_URL` and `FLEET_ENROLLMENT_TOKEN`)
+  
+```yaml
+- name: FLEET_URL
+  value: "https://123456.fleet.us-central1.gcp.cloud.es.io:443"
+  # Elasticsearch API key used to enroll Elastic Agents in Fleet (https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html#fleet-enrollment-tokens)
+  # If FLEET_ENROLLMENT_TOKEN is empty then KIBANA_HOST, KIBANA_FLEET_USERNAME, KIBANA_FLEET_PASSWORD are needed
+  - name: FLEET_ENROLLMENT_TOKEN
+  value: "WmpQbUVJVUJnX0kE6aDg0OGVtNG1TTDJqWlM2VGp"
+  ```
+
+2. [Elastic Agent Stanadlone Manifest]./manifests/elastic-agent-standalone-gke-autopilot.yaml)
+
+- Set Container Image version of Elastic Agent
+
+```yaml
+containers:
+    - name: elastic-agent-standalone
+      image: docker.elastic.co/beats/elastic-agent:8.5.3
+```
+
+- Provide Elastic Cloud Credentials
+  
+```yaml
+# This user needs the privileges required to publish events to Elasticsearch.
+- name: ES_USERNAME
+  value: "elastic"
+# The basic authentication password used to connect to Elasticsearch
+- name: ES_PASSWORD
+  value: "changeme"
+# The Elasticsearch host to communicate with
+- name: ES_HOST
+  value: "https://<url>.kb.us-central1.gcp.cloud.es.io:9243"
+- name: NODE_NAME
+```
+
+5. **Step 5: Install Elastic Agent Manifest:**
+
+For example:
+
+```bash=
+>kubectl apply -f elastic-agent-managed-kubernetes_gke_autopilot.yaml
+
+namespace/elastic-agent created
+configmap/agent-node-datastreams created
+Warning: Autopilot increased resource requests for DaemonSet elastic-agent/elastic-agent-standalone to meet requirements. See http://g.co/gke/autopilot-resources
+daemonset.apps/elastic-agent-standalone created
+clusterrolebinding.rbac.authorization.k8s.io/elastic-agent-standalone created
+rolebinding.rbac.authorization.k8s.io/elastic-agent-standalone created
+rolebinding.rbac.authorization.k8s.io/elastic-agent-standalone-kubeadm-config created
+clusterrole.rbac.authorization.k8s.io/elastic-agent-standalone created
+role.rbac.authorization.k8s.io/elastic-agent-standalone created
+role.rbac.authorization.k8s.io/elastic-agent-standalone-kubeadm-config created
+serviceaccount/elastic-agent-standalone created
+```
+
+> Elastic Agent is installed in namespace `elastic-agent`
+
+6. **Step 6: Install Kube-state metrics**
+
+> Kube-state Metrics needs to be installed in the same namespace as Elastic Agent
+
+```bash
+git clone git@github.com:kubernetes/kube-state-metrics.git
```

Comment:
I think is better to recommend install a stable version from the Helm Chart, that allows to set the namespace to install.

https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics

Reply:
done with latest commit